### ISSUE TITLE
Zoom doesnt work with simple approach

### ISSUE THREAD

**Issue by pnrobinson** [TheJacksonLaboratory/Gopher#163]
Caused by: java.lang.NullPointerException
	at gopher.model.viewpoint.ViewPoint.initRestrictionFragments(ViewPoint.java:240)
	at gopher.model.viewpoint.ViewPoint.init(ViewPoint.java:225)
	at gopher.model.viewpoint.ViewPoint.<init>(ViewPoint.java:176)
	at gopher.gui.viewpointpanel.ViewPointPresenter.zoom(ViewPointPresenter.java:504)
	at gopher.gui.viewpointpanel.ViewPointPresenter.zoomOut(ViewPointPresenter.java:533)
	... 82 more

**Comment by hansenp** [comment]
I just checked (develop branch). I get almost the same error message as shown above for the extended approach:
```
Caused by: java.lang.NullPointerException
	at gopher.model.viewpoint.ViewPoint.initRestrictionFragments(ViewPoint.java:240)
	at gopher.model.viewpoint.ViewPoint.init(ViewPoint.java:225)
	at gopher.model.viewpoint.ViewPoint.<init>(ViewPoint.java:176)
	at gopher.gui.viewpointpanel.ViewPointPresenter.zoom(ViewPointPresenter.java:508)
	at gopher.gui.viewpointpanel.ViewPointPresenter.zoomIn(ViewPointPresenter.java:529)
	... 82 more
```

**Comment by hansenp** [comment]
But it seems to work for the extended approach in branch viewpointTab.

**Comment by hansenp** [comment]
I will check this again after #167 is solved and everything is merged into develop.

**Comment by hansenp** [comment]
This issue was solved in branch `alignabilityMemorySaver`.
